src/components/PopupInfo/PopupInfo.jsx

```diff
@@ -1,43 +1,98 @@
+import { Box } from '@mui/material'
+import { approvalAction } from 'app/reducers/blog'
+import { SNACKBAR_SEVERITY, snackbarAction } from 'app/reducers/snackbar'
 import InforDetail from 'components/InforDetail/InforDetail'
 import { motion } from 'framer-motion'
+import useActions from 'hooks/useActions'
 import useShallowEqualSelector from 'hooks/useShallowEqualSelector'
 import { useState } from 'react'
+import { approvalBlog, rejectBlog } from 'rest/api/blog'
+import { parseImg } from 'utils'
 import * as styleMui from './PopupInfo.styled'
 
-function PopupInfo({ id }) {
+function PopupInfo({ id, approvalPage = false, setIndexData }) {
+    const { storeBlogSuccessfull } = useActions(approvalAction)
+    const { data: dataPlant } = useShallowEqualSelector((state) => state.plant)
+    const { data: dataApproval } = useShallowEqualSelector(
+        (state) => state.approval
```

Comment:
if you change name of slice reducer and name in store from `approval` to `blog`. Change here as well